### PR TITLE
using django-extensions to run initial data script

### PR DESCRIPTION
**Pull request by kave**
Realized the data migrations were becoming more & more problematic. Opted to go another route (http://django-extensions.readthedocs.org/en/latest/runscript.html?highlight=runscript)

Round 3 lets Go!

@rosskarchner 
@kurtw

### CODE REVIEW

File: cfgov/scripts/initial_data.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                              password=make_password(os.environ.get('WAGTAIL_ADMIN_PW')),
+                              is_superuser=True, is_active=True, is_staff=True)
+            admin_user.save()
+        # Renaming Site root name to `V1`
+        if not Page.objects.filter(title='V1').exists():
+            site_root = Page.objects.get(id=2)
+            site_root.title = 'V1'
+            site_root.save()
+        # Events Landing Page required for event `import-data` command
+        if not EventLandingPage.objects.filter(title='Events').exists():
```
Comment (kurtw) [inline cfgov/scripts/initial_data.py]: Comment from other PR

> The EventLandingPage type is deprecated in the BrowseFilterablePage PR. It's a normal BrowsePage now.

Comment (kave) [inline cfgov/scripts/initial_data.py]: @kurtw donzo

**Comment by kurtw** [comment]
So is the idea that you'd run this before `migrate` initially?

**Comment by kurtw** [comment]
Works just fine for me :+1:

**Comment by kave** [comment]
@kurtw nah migrate would run before this

**Comment by kurtw** [comment]
Gotcha. It's just `migrate` runs before the initial-data

**Pull request merged**